common/starbase_buildings/hfe_starbase_building_pirate.txt

```diff
@@ -9,6 +9,7 @@
 
 hfe_starbase_building_pirate_minerals = {
 	icon = GFX_spaceport_module_hydroponic_farms
+	initial = yes
```

Comment:
what mean initial?

Reply:
That means that it can be built at the start of the game (like shipyards) without any technology and without any exclusion.